Backup/backup-anywhere.md

```diff
@@ -1,7 +1,7 @@
 {{{
```

Comment:
Shouldn't API documentation be with all other API docs in https://www.ctl.io/api-docs/v2/? not distributed out by product in kbs

Reply:
I didn't realize that was there. That will need updated as well. I will add section(s) to that as well.

Reply:
I've created a pull request to add the API documentation for retrieving anywhere servers to https://github.com/CenturyLinkCloud/APIDocs as well